pkg/filesystem/local_directory_windows.go

```diff
@@ -584,44 +585,50 @@ func (d *localDirectory) Remove(name path.Component) error {
 	return err
 }
 
-// On NTFS mount point is a reparse point, no need to unmount.
 func (d *localDirectory) RemoveAllChildren() error {
+	return d.RemoveAllChildren(nil)
+}
+
+// On NTFS mount point is a reparse point, no need to unmount.
+func (d *localDirectory) RemoveAllChildren(dPath *path.Trace) error {
 	defer runtime.KeepAlive(d)
 
 	names, err := readdirnames(d.handle)
 	if err != nil {
-		return err
+		return util.StatusWrapf(err, "Failed to read contents of directory %#v", dPath.String())
 	}
 	for _, name := range names {
 		component := path.MustNewComponent(name)
 		fileType, err := d.lstat(component)
 		if err != nil {
-			return err
+			return util.StatusWrapf(err, "Failed to stat component %#v in %#v", name, dPath.String())
 		}
 		if fileType == FileTypeDirectory {
+			childPath := dPath.append(name)
 			subdirectory, err := d.enter(component, true)
 			if err != nil {
-				return err
+				return util.StatusWrapf(err, "Failed to enter component %#v in %#v", name, dPath.String())
 			}
-			err = subdirectory.RemoveAllChildren()
+			err = subdirectory.RemoveAllChildren(childPath)
 			subdirectory.Close()
 			if err != nil {
 				return err
 			}
 		}
 		err = d.Remove(component)
 		if err != nil {
-			return err
+			return util.StatusWrapf(err, "Failed to remove component %#v in %#v", name, dPath.String())
 		}
 	}
 	return nil
 }
 
 func (d *localDirectory) RemoveAll(name path.Component) error {
 	defer runtime.KeepAlive(d)
+	dPath = &path.Trace{}
```

Comment:
Just use `nil`. `nil` indicates the empty path (well, `"."`).

Reply:
Check :). I assumed it needed a kernel to start with, but the test looks to work with nil. And your listing seeds it with nil, rather than being a backward-compatible not-tracer.

Reply:
Done, in subsequent work.